docs/admins.rst

```diff
@@ -250,6 +250,20 @@ For example, you might want to set up an HTTP proxy:
   - name: NO_PROXY
     value: localhost,127.0.0.1
 
+Limiting image size
+'''''''''''''''''''
+
+It is doable to limit the binary image size before pushing to registry. If a
+binary image's size exceeds configured image size, it will be refused to be
+pushed. A typical configuration looks like::
+
+  "image_size_limit": {
+    "binary_image": 10000
+  }
```

Comment:
Nitpick: We could drop the brackets to match the other  reactor configmap examples in the docs. Moreover, when the top level section (where this subsection is contained) starts, it does not explicit mentions these entries are to be contained in the reactor configmap. We could make it a bit more explicit.

Not a blocker though.